taker/src/main.rs

```diff
@@ -308,7 +308,10 @@ async fn main() -> Result<()> {
     let figment = rocket::Config::figment()
         .merge(("address", opts.http_address.ip()))
         .merge(("port", opts.http_address.port()))
-        .merge(("cli_colors", false));
+        .merge(("cli_colors", false))
+        .merge(("secret_key", RandomSeed::default().seed()));
+    // in case we are implementing a session store - this seed should not be automatically generated
```

Comment:
🙃
I would prefer the log before the line

Reply:
Maybe this log is not even needed given we have the ticket https://github.com/itchysats/itchysats/issues/2978 already?